libs/blocks/share/share.js

```diff
@@ -76,39 +76,34 @@ export default async function decorate(block) {
         return {
           title: 'Facebook',
           href: `https://www.facebook.com/sharer/sharer.php?u=${url}`,
-          'daa-lh': 'facebook:click',
         };
       case 'twitter':
         return {
           title: 'Twitter',
           href: `https://twitter.com/share?&url=${url}`,
-          'daa-lh': 'twitter:click',
         };
       case 'linkedin':
         return {
           title: 'LinkedIn',
           href: `https://www.linkedin.com/sharing/share-offsite/?url=${url}`,
-          'daa-lh': 'linkedin:click',
         };
       case 'pinterest':
         return {
           title: 'Pinterest',
           href: `https://pinterest.com/pin/create/button/?url=${url}&description=${title}`,
-          'daa-lh': 'pinterest:click',
         };
       case 'reddit':
         return {
           title: 'Reddit',
           href: `https://reddit.com/submit?url=${url}&title=${title}`,
-          'daa-lh': 'reddit:click',
         };
       default:
         return null;
     }
   };
   if (!block.classList.contains('inline')) {
     const heading = toSentenceCase(await replaceKey('share-this-page', config));
-    block.append(createTag('p', null, heading));
+    block.append(createTag('p', { class: 'heading-title' }, heading));
```

Comment:
I see the table block has a similar class - https://github.com/search?q=repo%3Aadobecom%2Fmilo%20heading-title&type=code. I'm guessing this is just for having a consolidated selector, right?

Reply:
Maybe rename the class to something specific to analytics if it's only being used for that?

Reply:
Yes, same situation.  They didn't choose to use an actual header, but it was clear what was the "header" in terms of analytics.

Reply:
updated to tracking-header